src/containers/SecretsModal/SecretsModal.js

```diff
@@ -383,6 +414,11 @@ export /* istanbul ignore next */ class SecretsModal extends Component {
             handleAdd={this.handleAdd}
             invalidFields={invalidFields}
           />
+          <div
+            ref={el => {
```

Comment:
What's el?

Reply:
As far as I understand this line creates a reference to an element, so el would be the element created at the bottom of the modal